server/server.js

```diff
@@ -25,27 +25,45 @@ app.use(
         'default-src': ["'self'"],
         'connect-src': [
           "'self'",
-          'https://*.apps.gov.bc.ca',
           'https://orgbook.gov.bc.ca',
-          'https://*.oidc.gov.bc.ca',
+          'https://dev.oidc.gov.bc.ca',
+          'https://test.oidc.gov.bc.ca',
           'https://oidc.gov.bc.ca',
         ],
         'base-uri': ["'self'"],
         'block-all-mixed-content': [],
-        'font-src': ["'self'", 'https:', 'data:'],
+        'font-src': ["'self'"],
         'frame-ancestors': ["'self'"],
         'img-src': ["'self'", 'data:'],
         'object-src': ["'none'"],
-        'script-src': ["'self'", 'https://*.gov.bc.ca'],
+        'script-src': ["'self'"],
```

Comment:
this change looks good - we likely had the gov address enabled to support analytics, but we never ended up adding any gov analytics